src/js/index.js

```diff
@@ -76,4 +76,52 @@ form_submission.addEventListener('click', () =>{
     if(noEmptyForm > 2){
         warningText.classList.remove('form-warning-text');
     }
-});
\ No newline at end of file
+});
+
+//-----------------------------------------GALLERY------- (BLOCK - FEATURED WORKS)-----------------------------------------------------------------
+
+const closeGallery = document.querySelector('.modal-window-contain > .modal-btn-close');
+const windowGallery = document.querySelector('.modal-window-gallery');
+// btn closeGallery
+closeGallery.addEventListener('click', () =>{
+    windowGallery.style.display = 'none';
```

Comment:
add global class "g-hidden" with style {display: none}. 
It will be better to add class instead of adding style for the component.

`windowGallery.classList.add("g-hidden");`